src/viur/core/prototypes/list.py

```diff
@@ -190,7 +190,10 @@ def list(self, *args, **kwargs) -> t.Any:
                 if callable(default_order := self.default_order):
                     default_order = default_order(query)
 
-                if default_order:
+                if isinstance(default_order, tuple | list) and all(isinstance(pair, tuple) and len(pair) == 2
+                                                                   for pair in default_order):
+                    query.order(*default_order)
+                elif default_order:
                     query.order(default_order)
```

Comment:
Wait a moment... I think this is wrong, and your initial attempt is better. This is all ugly. A type should be better, as already mentioned here:
> I think when the Query class gets back to viur-core again, we should re-define the order-tuple to a namedtuple or dataclass, so that it has a clear definition.

Here's my alternative suggestion:
```suggestion
                if default_order:
                    if (
                        isinstance(default_order, str)
                        or (
                            isinstance(default_order, tuple)
                            and len(default_order) == 2
                            and isinstance(default_order[0], str)
                            and isinstance(default_order[1], db.SortOrder)
                        )
                    ):
                        query.order(default_order)
                    else:
                        query.order(*default_order)
```